org.nongnu.gsequencer.gsequencer.json

```
@@ -212,15 +254,32 @@
                 }
             },
             "post-install": [
+                "mkdir -p /app/share/doc/gsequencer/pdf/",
+                "install -c -p -m 644 pdf/user-manual-a4.pdf /app/share/doc/gsequencer/pdf/user-manual-a4.pdf",
+                "install -c -p -m 644 pdf/user-manual-letter.pdf /app/share/doc/gsequencer/pdf/user-manual-letter.pdf",
                 "sed -i -e 's~Icon=gsequencer~Icon=org.nongnu.gsequencer.gsequencer~g' /app/share/applications/gsequencer.desktop",
                 "install -c -p -m 644 COPYING /app/share/gsequencer/GPL-3",
```

I would remove the `shell` source below that does the exact same as here. (`post-install` is the righ way)
```suggestion
                "install -c -p -Dm644 COPYING -t /app/share/gsequencer/GPL-3",
```

---

Why -t?

---

maybe no `-t` here if you rename the file. I might have gone too far. Still the `-D` applies to make sure the directory exists.